dotcom-rendering/src/components/SignInGateSelector.importable.tsx

```diff
@@ -457,6 +450,45 @@ const fetchAuxiaDisplayDataFromProxy = async (
 	return Promise.resolve(data);
 };
 
+/*
```

Comment:
delete?

Reply:
Oh yeah done. Thanks! Another episode of talking to myself...